static/app/views/projectDetail/projectIssues.spec.tsx

```diff
@@ -4,8 +4,11 @@ import {render, screen, userEvent, waitFor} from 'sentry-test/reactTestingLibrar
 import ProjectIssues from 'sentry/views/projectDetail/projectIssues';
 
 describe('ProjectDetail > ProjectIssues', function () {
-  let endpointMock, filteredEndpointMock, newIssuesEndpointMock;
-  const {organization, router, routerContext} = initializeOrg({
+  let endpointMock: ReturnType<typeof MockApiClient.addMockResponse>,
```

Comment:
These are just `jest.Mock` types btw if you want to save yourself some keystrokes in the future  :)